subprocess/custom_exit.py

```diff
@@ -1,6 +1,8 @@
 import sys
 
+print(sys.argv)
+
 try:
-    raise SystemExit(sys.argv[1])
+    raise SystemExit(int(sys.argv[1]))
```

Comment:
:wrench: **Technical Recommendation**

Minor detail, but this line could technically raise a `ValueError` in case `argv[1]` is a non-integer value, such as `3.14` or `foobar`.

Reply:
Good catch!